redis/rediscache.go

```diff
@@ -0,0 +1,39 @@
+package rediscache
+
+import "github.com/garyburd/redigo/redis"
+
+// Cache is an implementation of httpcache.Cache that caches responses in a
+// redis server.
+type Cache struct {
+	redis.Conn
+}
+
+// cacheKey modifies an httpcache key for use in redis.  Specifically, it
```

Comment:
Idiomatic Go style and the style in this repo suggests to use a single space between sentences, not two.